provider/examples_test.go

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
+
+package provider
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+const examples_test_state_file = "test_examples.tfstate"
+
+var examples_test_allowed_environment_variables = []string{
```

Comment:
these const and var names should be mixed caps instead of underscores

Reply:
Fixed